src/utils/Router.js

```diff
@@ -31,7 +30,7 @@ export default function Routes() {
               <Route index element={<CreateViewSwitcher />} />
               <Route path=":id" element={<CertificateViewer />} />
             </Route>
-            <Route path="*" element={<Error404 />} />
+            <Route path="/certificates" element={<CertificatesViewAll />} />
```

Comment:
I removed this path intentionally before in #75. We should not have a `/certificates` path, the correct path is `/certificate`. Also we still need the 404 behavour

Reply:
Update coming shortly